### PR TITLE
Updating photo directory logic for future configs

### PR DESCRIPTION
**Pull request by radmint**
- Adding .vsconfig to .gitignore
- Removing /ExampleAssets/ folder and Photos.meta
- Adding file extension enum for configuring photo file type
- Adding more comments, standardizing file/photo/image usage in TakePic.cs

### CODE REVIEW

**Comment by croconut** [comment]
you'll need to assign the public variables in the player object to get it working in game. i would create a player prefab in a Prefab folder and connect everything thing that can be connected there. Scenes are hopefully just placements and connecting different prefabs together when they need external references. Extensions and stuff is pretty normal, the codes way more professional than i would make it lol 👍

File: Assets/Scenes/scene.unity [hunk 1]
```diff
@@ -1116,7 +1116,7 @@ Light:
   m_BounceIntensity: 1
   m_ColorTemperature: 6570
   m_UseColorTemperature: 0
-  m_BoundingSphereOverride: {x: 0, y: 0, z: 0, w: 0}
+  m_BoundingSphereOverride: {x: -1, y: 10, z: 11, w: -12}
```
Comment (radmint) [inline Assets/Scenes/scene.unity:1119]: Hmm I'm not sure what I did to incur this change. @croconut do you think I should just manual reset this line?

Comment (croconut) [reply Assets/Scenes/scene.unity:1119]: i think you need to set useBoundingSphereOverride to see a difference anyways

**Review by croconut — DISMISSED** [review]
lgtm

File: Assets/Scenes/scene.unity (hunk 1 above)
Comment (radmint) [reply Assets/Scenes/scene.unity]: Okay, I've gone ahead and reverted this in any case

**Review by croconut — APPROVED** [review]
okay so i forgot that this is putting files in the resources directory, we should definitely be looking into the application permanent data path for things that should be saved. would be a far better place to save, and it seems like it should be in this pr based on title
I am approving tho.

**Comment by radmint** [comment]
@croconut - done! Updated to use persistent store

**Pull request merged**